calculator/calculator.py

```diff
@@ -0,0 +1,25 @@
+def add(a, b):
+    return int(a) + int(b)
+def subtract(a, b):
+    return int(a) - int(b)
+def multiply(a, b):
+    return int(a) * int(b)
+def divide(a, b):
+    return int(a) / int(b)
+
+print("Select Operation:\n1- Add,\n2- Subtract,\n3- Multiply,\n4- Divide.")
+option = int(input("Enter you choice from 1/2/3/4 :- "))
```

Comment:
let's take the option as 2nd input.
I would like to have the flow like 2 + 3
For this, you might need to change the option checks to symbols like `+` `-` ... so on.